src/Sequel16.cpp

```diff
@@ -402,7 +402,7 @@ struct Sequel16 : Module {
 	dsp::PulseGenerator gatePulseR1;
 	dsp::PulseGenerator gatePulseR2;
 
-	SequelClockTracker clockTracker {16,3};
+	Sequel16ClockTracker clockTracker;
```

Comment:
in hindsight, the initialisation list wasn't the sole culprit of the issue and could still be used, however I think having two child classes like I've done here is neater anyway